app/messages.py

```diff
@@ -4,21 +4,22 @@
 NAME_INPUT_BY_USER_IS_INVALID = {"message": "Your name is invalid."}
 EMAIL_INPUT_BY_USER_IS_INVALID = {"message": "Your email is invalid."}
 USERNAME_INPUT_BY_USER_IS_INVALID = {"message": "Your username is invalid."}
-NEW_USERNAME_INPUT_BY_USER_IS_INVALID = {"message": "Your new username is" " invalid."}
+NEW_USERNAME_INPUT_BY_USER_IS_INVALID = {"message": "Your new username is invalid."}
 TOKEN_IS_INVALID = {"message": "The token is invalid!"}
 USER_ID_IS_NOT_VALID = {"message": "User id is not valid."}
-FIELD_NEED_MENTORING_IS_NOT_VALID = {"message": "Field need_mentoring is" " not valid."}
+FIELD_NEED_MENTORING_IS_NOT_VALID = {"message": "Field need_mentoring is not valid."}
 FIELD_AVAILABLE_TO_MENTOR_IS_INVALID = {
-    "message": "Field available_to_mentor" " is not valid."
+    "message": "Field available_to_mentor is not valid."
 }
 PASSWORD_INPUT_BY_USER_HAS_INVALID_LENGTH = {
     "message": f"The password field has to be at least {PASSWORD_MIN_LENGTH} characters and no more than {PASSWORD_MAX_LENGTH} characters."
 }
 PHONE_OR_MOBILE_IS_NOT_IN_NUMBER_FORMAT = {
-    "message": "Phone or mobile fields must be in number format. It must contain numbers and may contain dash ""-"" or space characters."
+    "message": "Phone or mobile fields must be in number format. It must contain numbers and may contain dash -"
```

Comment:
I think the older message description was correct.

Reply:
@Rahulm2310 They're the same, just reformatted (line 18+19)

Reply:
@NenadPantelic I meant to emphasize the `-` dash.